bin/spice/cmd/login.go

```diff
@@ -147,6 +147,36 @@ spice login s3 --access-key <key> --access-secret <secret>
 	}),
 }
 
+var postgresCmd = &cobra.Command{
+	Use:   "postgres",
+	Short: "Login to a Postgres instance",
+	Example: `
+spice login postgres --password <password>
+
+# See more at: https://docs.spiceai.org/
+`,
+	Run: CreateLoginRunFunc(api.AUTH_TYPE_POSTGRES, map[string]string{
+		passwordFlag: fmt.Sprintf("No password provided, use --%s or -p to provide a password", passwordFlag),
```

Comment:
If we support -p it seems we should also support -u for username?

Reply:
We don't currently support postgres username in secrets, but we could add that and then add to the CLI. Though it's not entirely necessary since usernames aren't sensitive.